### PR TITLE
qc views 302

### PR DESCRIPTION
**Pull request by briankip**
Why are you changing css/jquery.dataTables.css

### CODE REVIEW

**Comment by Piusyo** [comment]
@briankip kindly review and debug the part of returning the pre-entered results on the edit page.
I will meanwhile be working on the export of the 706 report.

**Comment by briankip** [comment]
Remove the .css

File: app/controllers/ControlController.php [hunk 1]
```diff
@@ -172,6 +172,48 @@ public function resultsEntry($controlId)
 		return View::make('control.resultsEntry')->with('control', $control)->with('lotNumber', $lotNumber)
 						->with('instrumentName', $instrumentName);
 	}
+	public function resultsshow($controlId)
+	{
+		$control = Control::find($controlId);
+		$controlTest = ControlTest:: all();
+		$lotNumber = Lot::where('instrument_id', $control->instrument_id)->orderBy('id', 'desc')->first()->number;
```
Comment (briankip) [inline app/controllers/ControlController.php]: What is the use of all this from 176 - 182

**Comment by Piusyo** [comment]
Is the fuction which populates of the resultsShow blade this the required information.

File: app/models/ControlTest.php [hunk 2]
```diff
@@ -11,5 +11,13 @@ public function controlResults()
 	{
 		return $this->hasMany('ControlMeasureResult');
 	}
+	public function Control()
```
Comment (briankip) [inline app/models/ControlTest.php]: All function must have documentation explaining what you are doing

**Comment by briankip** [comment]
Remove the lotNumber dropdown in the reports

**Comment by Piusyo** [comment]
It has been removed by reverting the above commit.

**Pull request merged**